src/layouts/MasterLayout.tsx

```diff
@@ -0,0 +1,15 @@
+import { Outlet } from "react-router-dom";
+import Navbar from "../components/Navbar";
+
+const MasterLayout = () => {
+  return (
+    <>
+      <Navbar />
+      <div className="master-layout bg-primaryLight dark:bg-primaryDark text-primaryDark dark:text-primaryLight">
```

Comment:
We can have a css variables in our css with the same name in different classes dark - light. and then use these color names in our tailwind classes. This will eliminate the need to write the classes manually for each element.

Reply:
Will search into that, because i dont understand what you have said 100% yet